devenv/log.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker logs -f $1
```

Comment:
another script for wiping `wipe` or `clean` could be useful for hard reset of the env.